mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosaPass.cpp

```diff
@@ -44,7 +45,8 @@ struct MIGraphXToTosa : public impl::MIGraphXToTosaPassBase<MIGraphXToTosa> {
     auto &ctx = getContext();
     RewritePatternSet patterns(&ctx);
     ConversionTarget target(ctx);
-    target.addLegalDialect<tosa::TosaDialect, func::FuncDialect>();
+    target.addLegalDialect<tosa::TosaDialect, func::FuncDialect,
```

Comment:
Why this?

Reply:
to run migraphx pipeline on the host for references for the clone test.

Reply:
Yeah, but you only need to mark a dialect "legal" if you emit it during the pass